sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/aio/_client.py

```diff
@@ -181,7 +181,7 @@ async def encrypt(self, algorithm: "EncryptionAlgorithm", plaintext: bytes, **kw
         if self._local_provider.supports(KeyOperation.encrypt, algorithm):
             raise_if_time_invalid(self._not_before, self._expires_on)
             try:
-                return self._local_provider.encrypt(algorithm, plaintext)
+                return self._local_provider.encrypt(algorithm, plaintext, iv=iv, aad=aad)
```

Comment:
If we need  `aad` only for AES-GCM, which we won't do locally, why pass it here?

Reply:
I decided to pass it since we'll ideally use it later, but it does make sense to only pass what's needed for now